gtk/MakeDialog.cc

```diff
@@ -347,12 +347,18 @@ void MakeDialog::Impl::updatePiecesLabel()
     }
     else
     {
-        gstr += gtr_sprintf(
-            ngettext("%1$s; %2$'d File", "%1$s; %2$'d Files", builder_->fileCount),
-            tr_strlsize(builder_->totalSize),
-            builder_->fileCount);
+        gstr += fmt::format(
+            ngettext("{complete_size}; {file_count:L} File", "{complete_size}; {file_count:L} Files", builder_->fileCount),
+            fmt::arg("complete_size", tr_strlsize(builder_->totalSize)),
+            fmt::arg("file_count", builder_->fileCount));
+
         gstr += "; ";
-        gstr += gtr_sprintf(
+
+        gstr += fmt::format(
+            ngettext("{piece_size}; {piece_count:L} Piece", "{piece_size}; {piece_count:L} Pieces", builder_->pieceCount),
```

Comment:
I agree, the earlier text was a littler clearer and it would be even better with more context.

33e5452075280c155db52a4e32103f9db88b1e71 has a revision that is a little more reasonable and shares more text between DetailsDialog and MakeDialog. If you're :+1: on this text I'd be fine with merging it, but I'm also open to further suggestions